test/cookbooks/minimal/recipes/default.rb

```diff
@@ -29,3 +29,28 @@
   name 'chef test'
   action :disable
 end
+
+windows_task 'create chef test' do
+  name 'chef\\chef test'
+  action :create
+  cwd "C:\\"
```

Comment:
Did you verify that cwd was actually being set?  I'm not seeing that working in my tests.